sdk/storage/azure-storage-blob/azure/storage/blob/_list_blobs_helper.py

```diff
@@ -110,49 +106,49 @@ def _extract_data_cb(self, get_next_return):
 
         return self._response.next_marker or None, self.current_page
 
-    def _build_item(self, item):
+    def _build_item(self, item: Union[BlobItemInternal, BlobProperties]) -> BlobProperties:
         if isinstance(item, BlobProperties):
             return item
         if isinstance(item, BlobItemInternal):
             blob = get_blob_properties_from_generated_code(item)  # pylint: disable=protected-access
-            blob.container = self.container
+            blob.container = self.container  # type: ignore [assignment]
```

Comment:
`azure\storage\blob\_list_blobs_helper.py:114: error: Incompatible types in assignment (expression has type "Optional[str]", variable has type "str")  [assignment]`